rpm/sailfish-browser.spec

```diff
@@ -117,11 +114,22 @@ install -m 644 -p %{SOURCE1} %{buildroot}%{_datadir}/mapplauncherd/privileges.d/
 
 # Upgrade, count is 2 or higher (depending on the number of versions installed)
 if [ "$1" -ge 2 ]; then
-%{_bindir}/add-oneshot --all-users --now browser-cleanup-startup-cache || :
```

Comment:
Imo with approach it'd be cleaner to keep this one as a oneshot.

Reply:
So let's keep "browser-cleanup-startup-cache" and schedule that for --all-users. If prefs and ua-updates do not exist those are/will be handled in the code.